Dockerfile

```diff
@@ -61,9 +61,9 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -ldflags "-s -
 
 FROM openjdk:11-jre-slim as release
 
-RUN apt-get update && \                                                                               
-    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y curl lsof net-tools && \ 
-    apt-get clean all                                                                                 
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y curl lsof net-tools && \
```

Comment:
do want to include apt-get ugprade?

Reply:
🤷 we're grabbing the latest jre image up there so is not going to be that much behind. The only concern would be image size. upgrade tends to blow out the image size.